### PR TITLE
feature(login_as): Added login_as plugin as bundled with the core.

### LINKED ISSUE #7958: Add login as functionality to core
Could simply adopt @brettp 's plugin for that https://github.com/brettp/Login-As

### PR DESCRIPTION
**Pull request by Srokap**
Fixes #7958

TODOs:
- [x] pick a more stable version of login_as to pull in, like `~1.0` (not `dev-master`)

### CODE REVIEW

**Comment by ewinslow** [comment]
dev-master sounds like a bad idea to me...

**Comment by Srokap** [comment]
Yes it does! 

I briefly talked today with @juho-jaakkola about asking @brettp to officially fork login_as to Elgg org and start properly versioning it. Perhaps @brettp can say here if it's fine with him.

Alternative would be to provide composer.lock, but just freezing on version is much more reliable. As running `composer update` would break composer.lock anyway.

File: engine/classes/ElggPluginManifest.php [hunk 1]
```diff
@@ -388,7 +388,7 @@ public function getCategories() {
 			'logbrowser', 'logrotate', 'members', 'messageboard',
 			'messages', 'notifications', 'pages', 'profile',
 			'reportedcontent', 'search', 'site_notifications','tagcloud', 'thewire',
-			'twitter_api', 'uservalidationbyemail', 'web_services', 'zaudio',
+			'twitter_api', 'uservalidationbyemail', 'web_services', 'zaudio', 'login_as',
```
Comment (ewinslow) [inline engine/classes/ElggPluginManifest.php]: Should be alpha-sorted

**Comment by Srokap** [comment]
I moved login_as plugin into Elgg and updated PR. If we want to keep bundled plugins ordered, we better have one entry per line. PTAL

File: composer.json [hunk 2]
```diff
@@ -17,7 +17,8 @@
         "knplabs/gaufrette": "~0.1.0",
         "tedivm/stash": "~0.12",
         "ircmaxell/password-compat": "~1.0",
-        "roave/security-advisories": "dev-master"
+        "roave/security-advisories": "dev-master",
+        "elgg/login_as": "1.9.1"
```
Comment (ewinslow) [inline composer.json]: Can you use ~1.9 here? That way we won't have to change it when login_as plugin is updated.

**Comment by Srokap** [comment]
PTAL

**Comment by ewinslow** [comment]
Awesometots!

**Pull request merged**